Source/Fuse.PushNotifications/JS.uno

```diff
@@ -125,5 +126,17 @@ namespace Fuse.PushNotifications
 			Fuse.PushNotifications.PushNotify.ClearAllNotifications();
 			return null;
 		}
+
+		/**
+			@scriptmethod register
+
+			Registers the app with APNS. Only neccesary if APNS.RegisterOnLaunch was
+			set to false in the unoproj file.
+		*/
+		public object Register(Context context, object[] args)
+		{
+			Fuse.PushNotifications.PushNotify.Register();
```

Comment:
for future-proofing it's best to check the number of arguments and issue UserError's if not 0.